2.cellprofiler_ic_processing/scripts/perform_ic.py

```diff
@@ -18,14 +18,13 @@
 import cp_utils as cp_utils
 import tqdm
 
-
 # ## Set paths
 
 # In[ ]:
```

Comment:
Recommend making sure this file is run sequentially until the last code cell prior to nbconvert.